src/core/shell_init.cpp

```diff
@@ -243,6 +244,47 @@ namespace mamba
             }
         }
     }
+
+    void enable_long_paths_support()
```

Comment:
really nice to have it, but same comment as for the header file this should be split in another PR